commands/operator-sdk/cmd/test.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2018 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+
+	cmdError "github.com/operator-framework/operator-sdk/commands/operator-sdk/error"
+	"github.com/spf13/cobra"
+)
+
+var (
+	testLocation     string
+	kubeconfig       string
+	namespace        string
+	crdManifestPath  string
+	opManifestPath   string
+	rbacManifestPath string
+	verbose          bool
+)
+
+func NewTestCmd() *cobra.Command {
+	testCmd := &cobra.Command{
+		Use:   "test --kubeconfig <path to kubeconfig> --image <name of operator image>",
```

Comment:
We don't need the operator --image flag anymore right?
Since we just use the input manifest to deploy the operator.